torch/autograd/gradcheck.py

```diff
@@ -112,16 +109,17 @@ def get_stride(size):
 
 
 def _get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3) -> List[Tuple[torch.Tensor, ...]]:
-    """Computes the numerical jacobian for a given fn and inputs. Returns M * N jacobians
-    where M is the number of input tensors that require grad, and N is the number of output
-    float/complex tensors.
+    """Computes the numerical Jacobian of `fn` at the provided input and with
+    respect to `target`, if provided. Returns M * N Jacobians where M is the number
```

Comment:
Interesting, the convention we usually use is M for the number of outputs and N for the number of inputs and a Jacobian of size MxN.
Is this comment actually accurate?

Reply:
Changed differentiable outputs -> non-integral outputs
input tensors -> tensors in `target`